WordPress/gutenberg
# Set default overlay color to black

## What?

Fixes https://github.com/WordPress/gutenberg/issues/49785

## Why?

Default overlay color isn't set in cover block, but we were loading black color though css, but the actual color picker shows nothing set.

## How?

Set default value to black, It will show black color selected in overlay color picker.

## Testing Instructions

1. Add cover block
2. Check overlay color in block setting
